client/overview/index.js

```diff
@@ -59,16 +59,20 @@ const OverviewPage = () => {
 		wpcomReconnectUrl,
 		featureFlags: { accountOverviewTaskList },
 	} = wcpaySettings;
-	const numDisputesNeedingResponse =
-		parseInt( wcpaySettings.numDisputesNeedingResponse, 10 ) || 0;
+
 	const { isLoading: settingsIsLoading, settings } = useSettings();
 
+	const { disputes: activeDisputes } = useDisputes( {
+		filter: 'awaiting_response',
```

Comment:
By default, we're fetching the disputes sorted by `created`. I think it's better if it's sorted by `due_by` since we're showing disputes by when they're due. What do you think?

I'm tempted to suggest adding a filter [here](https://github.com/Automattic/woocommerce-payments/blob/a0a1f9bd0f103b504508831d6994d347c148e06f/includes/core/server/request/class-list-disputes.php#L44-L53) so we can query disputes whose `due_by` within <= 7d but that's overengineering IMO.

```suggestion
		filter: 'awaiting_response',
		orderby: 'due_by',
```

Reply:
I take back my suggestion because I see that the `due_by` field can be null and not indexed!